packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.Props.ts

```diff
@@ -0,0 +1,107 @@
+import { Position } from '../../utilities/positioning';
+import { IIconProps } from '../../Icon';
+
+export interface ISpinButtonProps {
+
+  /**
+   * The initial value of the SpinButton. Use this if you intend for the SpinButton to be an uncontrolled component.
+   * This value is mutually exclusive to value. Use one or the other.
+   * @default 0
+   */
+  defaultValue?: string;
```

Comment:
This is also missing _warnMutuallyExclusive for defaultValue and value in the component constructor